src/rospkg/os_detect.py

```diff
@@ -126,6 +139,47 @@ def get_codename(self):
             return self.lsb_info[2]
         raise OsNotDetected('called in incorrect OS')
 
+class FdoDetect(OsDetector):
```

Comment:
Ahh, that makes sense now. I was reading it as an oddly truncated fedora. Maybe you could put the link to the spec into the comment since that's what people in the future will see anyway.